gunicorn/workers/base.py

```diff
@@ -156,7 +155,7 @@ def load_wsgi(self):
                 exc_type, exc_val, exc_tb = sys.exc_info()
                 self.reloader.add_extra_file(exc_val.filename)
 
-                tb_string = traceback.format_tb(exc_tb)
+                tb_string = traceback.format_exc(exc_tb)
```

Comment:
Why did you change this? The usage of `format_tb()` was discussed in https://github.com/benoitc/gunicorn/pull/1235#issuecomment-201002065.